src/masternodes/govvariables/attributes.cpp

```diff
@@ -1268,6 +1268,11 @@ Res ATTRIBUTES::Apply(CCustomCSView & mnview, const uint32_t height)
                 }
             } else if (attrV0->key == TokenKeys::LoanCollateralFactor) {
                 if (height >= static_cast<uint32_t>(Params().GetConsensus().FortCanningEpilogueHeight)) {
+                    // Skip on JellyFish
+                    if (gArgs.GetBoolArg("-regtest-skip-loan-collateral-validation", false)) {
```

Comment:
1. Add a check to ensure it's first REGTEST network and chain it with the arg after. 
2. Remove the Skip on jellyfish comment.